### ISSUE TITLE
Could I know the experimental environment (parameters) when fine tuned?

### ISSUE THREAD

**Issue by JongBeomBaek** [nianticlabs/monodepth2#312]
I tried learning your code with the default option. (resnet 18 & resnet 50,  pytorch version 0.4.1 & 1.7) 
but the performance was not the same as the result in your paper. (loss did not fall)

I would appreciate it if you let me know the environmental conditions when fine-tuned.

**Comment by JongBeomBaek** [comment]
> I tried learning your code with the default option. (resnet 18 & resnet 50, pytorch version 0.4.1 & 1.7)
> but the performance was not the same as the result in your paper. (loss did not fall)
> 
> I would appreciate it if you let me know the environmental conditions when fine-tuned.
